### DISCUSSION TITLE
Server Settings Wishlist

### DISCUSSION THREAD

**Discussion by gramian** [ArcadeData/arcadedb#595] [Ideas]
Dear Developers,

I really like the clean project structure of ArcadeDB; so, as I can configure the location of the database as commandline argument to the server via [`server.databaseDirectory`](https://docs.arcadedb.com/#Settings), I would like to be able to pass target paths for backups and logs with server settings like: 
 
* `server.backupDirectory` (while triggering a backup via SQL is cool, passing the path as SQL argument seems impractical)
* `server.logDirectory`

Furthermore, it would be easier with secrets management (see [docker](https://docs.docker.com/engine/swarm/secrets/) and [kubernetes](https://kubernetes.io/docs/concepts/configuration/secret/)) if the root password could be passed as a path to a file holding it, instead of directly as string, hence I would like a server setting like:

* `server.rootPasswordPath` 

I understand that these new settings may not match the project philosophy.

Thank you for considering

**Comment by lvca** [comment]
I like them all, creating an issue from this discussion.
